package.json

```diff
@@ -11,13 +11,16 @@
     "teachcode": "./bin/index.js"
   },
   "scripts": {
-    "lint": "eslint src bin test",
-    "lint:fix": "eslint --fix src bin test",
+    "lint": "eslint src bin __e2e__",
+    "lint:fix": "eslint --fix src bin __e2e__",
     "pretest": "npm run lint",
-    "test": "ava test/index.js",
+    "test": "ava --serial",
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs"
   },
+  "ava": {
+    "snapshotDir": "__snapshots__"
```

Comment:
```suggestion
    "snapshotDir": "__e2e__/__snapshots__"
```

Reply:
Doing this gives the following directory structure:
--- \_\_e2e\_\_/
    |---------cli.test.js
    |---------commands/
    |---------\_\_snapshots\_\_/
        |------------\_\_e2e\_\_/
            |--------------cli.test.js.md
            |--------------cli.test.js.snap
            |--------------commands/
                     |-----------------showkeys.test.js.md
                     |-----------------showkeys.test.js.snap
                     |-----------------showcommands.test.js.md
                     |-----------------showcommands.test.js.snap

But I don't think this is what we desire.

Reply:
Everything related to `e2e tests` should come under the `__e2e__` directory.